C#HomeWork/Hash&Salt/Authen.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Security.Cryptography;
+
+public class Authen
+{
+    private List<string> usernames = new List<string>();
+    private List<string> passwords = new List<string>();
```

Comment:
có thể dùng List<Account> hoặc Dictionary<string, string>, việc để 2 collection làm chung 1 việc này sẽ mất thời gian và làm cho code rối

```suggestion
    private List<Account> Accounts= new List<Account>();
```